admin-dev/themes/new-theme/js/pages/order/create/create-order-page.js

```diff
@@ -128,6 +142,8 @@ export default class CreateOrderPage {
     this.$container.on('blur', createOrderMap.cartRuleSearchInput, () => this.cartRuleManager.stopSearching());
     this._listenForCartEdit();
     this._onCartLoaded();
+    this._onCustomersNotFound();
```

Comment:
```suggestion
    this.onCustomersNotFound();
```
Please don't use underscore :)